docs/user/setup.rst

```diff
@@ -110,6 +110,53 @@ installation of FreeBSD.
     $ wget https://raw.githubusercontent.com/scala-native/scala-native/master/scripts/scala-native.nix
     $ nix-shell scala-native.nix -A clangEnv
 
+**Windows**
+
+There are perhaps many different possible ways to setup an environment on
+Windows but this way works and is not too difficult. Powershell was used
+for installing and running Scala Native.
+
+1. Download and install Visual Studio Community 2019
```

Comment:
Hi, this could be replaced with chocolatey or scoop, eg: https://community.chocolatey.org/packages/visualstudio2019community


Similarly with LLVM:

https://community.chocolatey.org/packages/llvm

Reply:
> Hi, this could be replaced with chocolatey or scoop, eg: https://community.chocolatey.org/packages/visualstudio2019community
> 
> Similarly with LLVM:
> 
> https://community.chocolatey.org/packages/llvm

I agree. Alternatively a full command line with appropriate arguments could be also useful.
Example:
```
".\vs_community__<version>.exe" --add Microsoft.VisualStudio.Component.PortableLibrary 
    --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 
    --add Microsoft.VisualStudio.Component.Windows10SDK.18362 
    --quiet --wait --norestart --noUpdateInstaller
```

Reply:
I will clean this up and mention chocolatey and scoop as alternatives. I needed to install the way I did because of corporate constraints which I will also mention. The last command line example is more complicated so I don't really want to add that although it seems more direct. BTW, I am a Windows novice so I'm sure many people have much fancier ways to go about this. We can always update the instructions after this PR based on feedback. Right now I am trying to fix the problem and wanted to add some docs that worked for me in a corporate environment.

I was actually surprised it would be this easy and it would just work. :smile: